integrations/integration_cname.js

```diff
@@ -42,6 +42,7 @@ const commonNames = {
   FULLSTORY: "FULLSTORY",
   Fullstory: "FULLSTORY",
   BUGSNAG: "BUGSNAG",
+  TVSQUARED: "TVSQUARED",
```

Comment:
also put corresponding entry in `client_server_name.js`

Reply:
integrations/client_server_name.js

I have put here..

Reply:
ohh... missed it.